data/transactions/logic/assembler.go

```diff
@@ -415,7 +419,7 @@ func assembleLoad(ops *OpStream, args []string) error {
 	if err != nil {
 		return err
 	}
-	if val > 255 {
+	if val > EvalMaxScratchSize {
 		return errors.New("load limited to 0..255")
```

Comment:
move this to file level, use it here and in store err
`var errScratchOutOfBounds = fmt.Errorf("scratch space limited to 0..%d", EvalMaxScratchSize)`